src/test/add-remove.test.js

```diff
@@ -0,0 +1,34 @@
+import { displayTasks, removeTask } from '../modules/utils';
+import Task from '../modules/tasks';
+
+const Tasks = new Task();
+
+document.body.innerHTML = '<ul class="nav__items"></ul>';
+const listContainer = document.querySelector('.nav__items');
+
+describe('Test Add Task and local storage', () => {
+  window.localStorage = Storage.prototype;
+  test('Add new Task', () => {
+    const task = {
+      index: 0,
+      description: 'This is sample test',
+      completed: false,
+    };
+    Tasks.addTask(task);
+    displayTasks(Tasks.tasksArray, listContainer);
+    expect(Tasks.tasksArray).toHaveLength(1);
+  });
+
+  test('Test local storage', () => {
+    JSON.parse(localStorage.getItem('todos'));
+    expect(localStorage).toHaveLength(1);
+  });
+});
```

Comment:
- [ ] It looks like your tests are not passing. There is a note saying that `local storage` is not defined. I would advise you to use mocks that imitate the functioning of the local storage and by doing this, you will be able to make your tests pass. Here is a [document](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/) to help you learn more about mocks. Also this [post](https://stackoverflow.com/questions/32911630/how-do-i-deal-with-localstorage-in-jest-tests) on stack overflow. 😺 
![image](https://user-images.githubusercontent.com/90222110/178585921-e9b25d57-4bf6-4992-888e-66efd95e6fa1.png)